data/mesongenbuildreq.py

```diff
@@ -2,8 +2,11 @@
 import json
 import sys
 deps_json = json.loads(subprocess.run([sys.argv[1], "introspect", "--dependencies", "meson.build"], capture_output=True).stdout)
-deps = dict(sorted(zip([x['name'] for x in deps_json],[x['version'] for x in deps_json])))
-deps.pop('', None)
+unsorted_deps = dict(zip([x['name'] for x in deps_json],[x['version'] for x in deps_json]))
```

Comment:
While you're changing this, can you add a space after the coma? `deps_json], [x['version']`